### ISSUE TITLE
RNKeychainManager.hasInternetCredentialsForServer was called with 2 arguments but expects 1 argument

### ISSUE THREAD

**Issue by vascofg** [oblador/react-native-keychain#306]
Getting this error on iOS with version 5.0.0 of the library.
It seems this method was updated from 4.0.5 on the Javascript side but not on the iOS side.

**Comment by vascofg** [comment]
@oblador @OleksandrKucherenko PR created

**Comment by etaiklein** [comment]
I received something similar to this: https://github.com/oblador/react-native-keychain/issues/212
Could there be a similar solution as well?

**Comment by vascofg** [comment]
Don't think so @etaiklein, since that call on Javascript actually matches the number of arguments on the iOS code:
```js
return RNKeychainManager.setGenericPasswordForOptions(
    options,
    username,
    password
  );
```

matches this one (plus promise handlers)

```objective-c
RCT_EXPORT_METHOD(setGenericPasswordForOptions:(NSDictionary *)options
                  withUsername:(NSString *)username
                  withPassword:(NSString *)password
                  resolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject)
```

**Comment by oblador** [comment]
Fixed in [5.0.1](https://github.com/oblador/react-native-keychain/releases/tag/v5.0.1).

**Comment by cladjules** [comment]
Seems like it's broken again in 6.0.0
